tools/build_all.sh

```diff
@@ -7,11 +7,13 @@
 # set at the time we start the last step, the ros1_bridge build.
 
 if [ "$ROS2_DISTRO" = "" ]; then
```

Comment:
Please revert to 4 spaces. Its customary for shell script.

Reply:
Do you find weird that after this block every indentation is 2 space indentation?

Reply:
Ah, that is strange. Lets just make it consistent, whether 4 or 2 I suppose.